### PR TITLE
Cookie Count Stat Reset now independent of Activity Log Removals. 

### LINKED ISSUE #777: Clearing Logs delete a number of all deleted cookies.
**Describe the bug**

Clearing Logs, shouldn't delete the number of cookies, which the Cookie AutoDelete has deleted.

**To Reproduce**

1. Go to 'Open settings'
2. Click on 'Cleanup Log'
3. Click on 'Clearing Logs'
4. See that the total amount of deleted cookies is 0, instead of the number, of deleted cookies, which you had before.

**Expected behavior**

Clicking on Cleanup Log, and then on Clearing Logs, should remove the logs, not the total amount of deleted cookies.

**Screenshots**

 - OS: [Windows 7 Pro]
 - Browser Info: [Firefox 77.0.1]
 - CookieAutoDelete Version [3.3.1]

**Additional context**

I don't know why there is the **untested bug/issue**, since it's a very easily reproducible bug.

### PR DESCRIPTION
**Pull request by kennethtran93**
Closes #777.
- Adds a button on welcome page to reset counters.
- Clearing Activity Logs no longer clear cookie count stats.
- fixes some eslint warnings and refactoring codes from modified files.

Signed-off-by: Kenneth Tran <kennethtran93@users.noreply.github.com>

### CODE REVIEW

**Comment by kennethtran93** [comment]
@ke-d  It's not really a large commit...the majority of the diff lines are in locale messages.json after refactoring it, but the only change to that local file was add a translation in that file.

**Comment by kennethtran93** [comment]
Apologies for all the close and re-open - seems the GitHub Actions that I added had some minor bugs in it that I didn't catch earlier.

File: src/ui/settings/ReleaseNotes.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": [
+    {
+      "version": "unreleased-alpha",
```
Comment (ke-d) [inline src/ui/settings/ReleaseNotes.json:4]: Are you changing this in the future? What I do is I set the next version here so I don't forget before releasing.

**Review by ke-d — APPROVED** [review]

File: src/ui/settings/ReleaseNotes.json (hunk 1 above)
Comment (kennethtran93) [reply src/ui/settings/ReleaseNotes.json:4]: I usually set it when I bump for release depending on what has been done - so next could be either 3.5.0 or 3.4.1.

**Pull request merged**